src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -46,18 +46,25 @@ <h5>{{ version }}</h5>
              i18n-aria-label
              id="clusterStatus">
       <div class="d-flex flex-column justify-content-center align-items-center">
-        <ng-template #healthChecks>
-          <cd-health-checks [healthData]="healthData.health.checks"></cd-health-checks>
-        </ng-template>
+
         <ng-template #healthWarningAndError>
-        <div class="info-card-content-clickable mt-1"
+        <!-- <div > -->
+          <cds-toggletip
              [ngStyle]="healthData.health.status | healthColor"
-             [ngbPopover]="healthChecks"
-             popoverClass="info-card-popover-cluster-status">
+             [dropShadow]="true"
+             placement="bottom"
+             class="info-card-content-clickable mt-1">
```

Comment:
This class is for info-card, which seems like the old popover element.
Why did not we chnaged that as per carbon and used here ?